README updated.txt

```diff
@@ -0,0 +1,34 @@
+# Metriq API Client
```

Comment:
The README file is helpful, but, in general, please keep pull requests limited to single topics or issue tickets. Also, for example, so that these two pieces could be reviewed separately, you would branch them both independently from the head of `main` branch. GitHub would handle the merge, just fine.

Reply:
On it, git automatically merged them two together but I'm trying to figure out now how to revert the read me commit so it's is just the task.py like how Nathan suggested.

Reply:
No problem. We'll go over this tomorrow after the weekly meeting.